### PR TITLE
Doc iteration userguide

### PR DESCRIPTION
**Pull request by LocoRichard**

### CODE REVIEW

**Review by claireyuw — CHANGES_REQUESTED** [review]

File: site/en/home/home.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ id: home.md
 
 # Welcome to Milvus Docs!
 
-Here you will learn about [what Milvus is](overview.md), and how to [install](install_standalone-docker.md), [use](connect.md), and [deploy](aws.md) Milvus to build an [application](image_similarity_search.md) according to your business need. You will also find [FAQs](performance_faq.md) and [API references](https://milvus.io/api-reference/pymilvus/v2.0.0rc7/api/collection.html) here.
+
+Here you will learn about [what Milvus is](overview.md), and how to [install](install_standalone-docker.md), [use](manage_connection.md), and [deploy](aws.md) Milvus to build an [application](image_similarity_search.md) according to your business need. You will also find [FAQs](performance_faq.md) and [API references](https://milvus.io/api-reference/pymilvus/v2.0.0rc7/api/collection.html) here.
```
[the remark this answers is not in the document]
Comment (LocoRichard) [reply site/en/home/home.md:8]: Angel said that the front end cannot render variables in this doc. Changing it to rc8

[the remark this answers is not in the document]
Comment (LocoRichard) [reply site/en/home/home.md:8]: Actually, I'll leave it to angela. This page is still about rc7, and I have no idea which must be be change. @YiyunNi

File: site/en/userGuide/manage_data.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+```
+
+
+
+## Delete entities
+
+Milvus 2.0 supports deleting entities by primary key specified with boolean expression.
+
+<div class="alert note">
+  Current release of Milvus only supports deleting entities by primary key.
```
[the remark this answers is not in the document]
Comment (LocoRichard) [reply site/en/userGuide/manage_data.md:136]: those are a note followed by a caution.

[the remark this answers is not in the document]
Comment (claireyuw) [reply site/en/userGuide/manage_data.md:136]: Same rule. Note, caution, and warning are all notices.

File: site/en/userGuide/manage_index.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
+id: manage_index.md
+related_key: create index
+summary: Learn how to build an index for vectors in Milvus.
+---
+
+# Manage Indexes
+
+This topic describes how to manage indexes in Milvus. See [Vector Index](index.md) and [Index Selection](index_selection.md) for more information.
```
[the remark this answers is not in the document]
Comment (LocoRichard) [reply site/en/userGuide/manage_index.md:9]: Done. btw, you only need to specify a boolean expression during a query, which is essentially a filtering operation on the scalar field or pk field.

**Review by claireyuw — CHANGES_REQUESTED** [review]

**Comment by claireyuw** [comment]
@LocoRichard Review is done except for `search.md`, `query.md`, and `hybridsearch.md`. These three need further discussion.

Please make changes to the rest of the files accordingly and apply those to CN files as well.

**Pull request merged**